homework/english/svet_konduhowich/lecture1/task1.md

```diff
@@ -0,0 +1,47 @@
+# Homework_Englich_ Last try.
+1. Is your mum working today?
+2. Why are these boys talking now?
+3. Where does he live?
+4. Does their mother cook very well?
+5. Lili is not rich, she doesn't drive a Mercedes.
+6. Let´s go out. It is not raining now.
+7. I have to go now. It is getting dark.
+8. A young man usually stands at the window.
+9. Ann usually makes up doll dresses herself.
+10. Is his wife having coffee at the moment?
+11. Let’s go for a walk. It isn't raining.
+12. Look! The boys are coming home.
+13. Does Maila like Chinese food?
+14. My grandmother doesn't work. She is on pension.
+15. What is your brother watching now?
+16. Where do you live?
+17. Do you play the piano well?
+18. Where are their brothers swimming now?
+19. What do them eating at breakfast?
```

Comment:
they

Reply:
I think it is their i right

Reply:
it seems must be a subject(i, you, he, she, it, we, they)
they do